tests/test_model.py

```diff
@@ -5955,9 +6100,9 @@ def test_explain(self):
              'Protocol: "Overdrive"',
              'Used by library: "The only library"',
              'External account ID: "id"',
+             'Setting "setting": "value"',
              'Setting "url": "url"',
              'Setting "username": "username"',
-             'Setting "setting": "value"'
```

Comment:
This test started failing on my machine because items started coming out differently from a hash. I changed explain() to sort the keys.